Control/lib/services/Environment.service.js

```diff
@@ -55,12 +57,30 @@ class EnvironmentService {
    * @throws {Error}
    */
   async getEnvironment(id, taskSource) {
-    const {environment} = await this._coreGrpc.GetEnvironment({id});
-    const detectorsAll = this._apricotGrpc.detectors ?? [];
-    const hostsByDetector = this._apricotGrpc.hostsByDetector ?? {};
-    const environmentInfo = EnvironmentInfoAdapter.toEntity(environment, taskSource, detectorsAll, hostsByDetector);
+    try {
+      const {environment} = await this._coreGrpc.GetEnvironment({id});
+      const detectorsAll = this._apricotGrpc.detectors ?? [];
+      const hostsByDetector = this._apricotGrpc.hostsByDetector ?? {};
+      const environmentInfo = EnvironmentInfoAdapter.toEntity(environment, taskSource, detectorsAll, hostsByDetector);
+      return environmentInfo;
```

Comment:
Return directly here

Reply:
Fixed in [2205](https://github.com/AliceO2Group/WebUi/pull/2205)